### ISSUE TITLE
Does Chrome Canary and Safari not work with KeepassXC?

### ISSUE THREAD

**Issue by intradesign** [keepassxreboot/keepassxc#6178]
Hi,
as al longtime 1password user I just stumbled upon KeepassXC and I am really impressed of the development compared to older versions.

I was able to connect the KeepassXC database to Firefox, Firefox Dev Edition and Chrome but I was not able to connect it to Chrome Canary. Also there seems to be no addon for safari.

Is there any way to connect these browsers?

Thanks and best regards, Timo

KeePassXC - unknown (because not connected to the database; it is the latest version)
KeePassXC-Browser - 1.7.6
Operating system: MacIntel (Big Sur)
Browser: Chrome/Chromium 90.0.4426.2 (it is Chrome Canary)

**Comment by droidmonkey** [comment]
For Chrome canary you might need to define a custom browser. Go to the application settings -> Browser Plugin -> Advanced

We do not support Safari yet due to technical reasons. https://github.com/keepassxreboot/keepassxc-browser/issues/294

**Comment by intradesign** [comment]
Hi and thanks for your reply.

I have tried this and it worked somehow but the connection between
browser(s) (any browser btw.) and Keepass suddenly breaks and I always get
error messages and have to restart the browser and/or Keepass. This is very
inconvenient.

So after giving Keepass a try I came to the conclusion that it
cannot replace 1Password.

Am Mi., 24. Feb. 2021 um 13:28 Uhr schrieb Jonathan White <
notifications@github.com>:
